docs/getting_started/creating_a_flyte_project.md

```diff
@@ -0,0 +1,79 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+# Creating a Flyte project
```

Comment:
I'm torn between introducing git... etc. here vs jumping right to running things...